lib/batch_message.js

```diff
@@ -0,0 +1,26 @@
+function Batch () {
+	var messages = Array.prototype.slice.call(arguments)
+
+	this.op = 'batch'
+	this.value = messages
+}
+
+Object.defineProperty(Batch.prototype, 'length', {
```

Comment:
is this supported on browsers we need? https://support.zendesk.com/hc/en-us/articles/203661786-Zendesk-system-requirements